packages/react/src/components/PageTitleBar/PageTitleBar.story.jsx

```diff
@@ -145,13 +131,13 @@ export const WithStatusDescriptionAndCrumbs = () => (
       headerMode={select('headerMode', ['DYNAMIC', 'STATIC', 'STICKY', 'CONDENSED'], 'STATIC')}
     />
   </div>
-);
+));
 
 WithStatusDescriptionAndCrumbs.story = {
   name: 'with status description and breadcrumbs',
 };
 
-export const WithEditableTitleBar = () => (
+export const WithEditableTitleBar = withReadme(README, () => (
```

Comment:
This one is a knob in the other stories, BUT the action callback is not working. Probably related to the wrapper function again. We should try to find the root cause of this...

Reply:
Yeah, this is the exact same thing as the `onToggleAggregations` callback or button `onClick` callback. It works locally and in next, but fails in these storybook netlify deploys. I think it has to be related to netlify, but it even fails if you're not opening it within the storybook frame, but opening the [iframe directly](https://deploy-preview-2359--carbon-addons-iot-react.netlify.app/iframe.html?id=1-watson-iot-pagetitlebar--with-editable-title-bar&args=). 🤷‍♂️